### PR TITLE
#AB89911 Endpoint to create consortium

### PR DESCRIPTION
**Pull request by suskiwojciech**
### 🛠 Changes being made

Summarize the changes you've made in this pull request.

### 📸 Screenshots (optional)

Insert some screenshots if you made UI changes.

### 🏎 Quality check

- [ ] Have you performed local tests?
- [ ] Are integration tests passing locally?
- [ ] Have you added some unit tests?
- [ ] Have you added some integration tests?
- [ ] Have you checked WCAG (included screenshot)

### CODE REVIEW

File: src/HE.CRM.Common/DtoMapping/ConsortiumMapper.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+using Microsoft.Xrm.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Web.UI;
+
+namespace HE.CRM.Common.DtoMapping
+{
+    public class ConsortiumMapper
+    {
+        public static ConsortiumDto MapReguralEntityToDto(invln_Consortium consortium, List<invln_ConsortiumMember> consortiumMember)
```
Comment (jjankowski87) [inline src/HE.CRM.Common/DtoMapping/ConsortiumMapper.cs:12]: Regural -> Regular

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investments.Common.CRM/Model/Actions/invln_setconsortium.cs [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	[Microsoft.Xrm.Sdk.Client.RequestProxyAttribute("invln_setconsortium")]
+	public partial class invln_setconsortiumRequest : Microsoft.Xrm.Sdk.OrganizationRequest
+	{
+		
+		public static class Fields
+		{
+			public const string invln_userid = "invln_userid";
+			public const string invln_programmeId = "invln_programmeId";
+			public const string invln_consortiumname = "invln_consortiumname";
+			public const string invln_leadpartner = "invln_leadpartner";
```
Comment (Carq) [inline src/HE.Investments.Common.CRM/Model/Actions/invln_setconsortium.cs:25]: `invln_leadpartner` -> `invln_leadpartnerId` ?

File: src/HE.Investments.Common.CRM/Model/Actions/invln_getconsortium.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+	
+	
+	[System.Runtime.Serialization.DataContractAttribute(Namespace="http://schemas.microsoft.com/xrm/2011/new/")]
+	[Microsoft.Xrm.Sdk.Client.RequestProxyAttribute("invln_getconsortium")]
+	public partial class invln_getconsortiumRequest : Microsoft.Xrm.Sdk.OrganizationRequest
+	{
+		
+		public static class Fields
+		{
+			public const string invln_consortiumid = "invln_consortiumid";
```
Comment (Carq) [inline src/HE.Investments.Common.CRM/Model/Actions/invln_getconsortium.cs:22]: I have added to task another field `memberOrganisationId`. CRM should check if organisation member is part of given consortiumId .

Comment (suskiwojciech) [reply src/HE.Investments.Common.CRM/Model/Actions/invln_getconsortium.cs:22]: Added

File: src/HE.Investments.Common.CRM/Model/Actions/invln_setconsortium.cs (hunk 2 above)
Comment (suskiwojciech) [reply src/HE.Investments.Common.CRM/Model/Actions/invln_setconsortium.cs:25]: fixed

**Review by Carq — APPROVED** [review]

**Review by Carq — APPROVED** [review]

**Pull request merged**